core/src/main/scala/org/http4s/StaticFile.scala

```diff
@@ -1,18 +1,21 @@
 package org.http4s
 
+import java.io._
+import java.net.URL
+
+import cats.Semigroup
```

Comment:
We generally have all imports in the same list, ordered alphabetically.

Reply:
We ended up removing the IntelliJ setting that did this for people, right?  It's a pity that scalafmt can't just do this for people.

Reply:
Yeah, sorry, I was using the default IntelliJ settings. This should be resolved now!